File: Desafios em C#/2748 - Saída 2.cs
//Developed by: @LucasMarcuzo

using System; 

class URI 
{
    static void Main(string[] args) 
    {   

        Tracos();

        Console.WriteLine("");
        for (int i = 0; i < 39; i++) {
            if(i == 0 || i == 38){
                Console.Write("|");
            }
            else if(i == 10){
                Console.Write("        Roberto                      ");
            }
        }

        Espaco();

        Console.WriteLine("");
        for (int i = 0; i < 39; i++) {
            if(i == 0 || i == 38){
                Console.Write("|");
            }
            else if(i == 10){
                Console.Write("        5786                         ");
            }
        }

        Espaco();

        Console.WriteLine("");
        for (int i = 0; i < 39; i++) {
            if(i == 0 || i == 38){
                Console.Write("|");
            }
            else if(i == 10){
                Console.Write("        UNIFEI                       ");
            }
        }

        Console.WriteLine("");
        Tracos();
        Console.WriteLine("");

    }

    private static void Espaco(){
        Console.WriteLine("");
        for (int i = 0; i < 39; i++) {
            if(i == 0 || i == 38){
                Console.Write("|");
            }
            else{
                Console.Write(" ");
            }   
        }
    }

    private static void Tracos(){
        for (int i = 0; i < 39; i++) {
            Console.Write("-");
        }
    }
}

File: Desafios em JavaScript/2748 - Saída 2.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var aux = "";

tracos();

aux += ("\n");
for (let i = 0; i < 39; i++) {
    if(i === 0 || i === 38){
        aux += ("|");
    }
    else if(i == 10){
        aux += ("        Roberto                      ");
    }
}

espaco();

aux += ("\n");
for (let i = 0; i < 39; i++) {
    if(i === 0 || i === 38){
        aux += ("|");
    }
    else if(i == 10){
        aux += ("        5786                         ");
    }
}

espaco();

aux += ("\n");
for (let i = 0; i < 39; i++) {
    if(i === 0 || i === 38){
        aux += ("|");
    }
    else if(i == 10){
        aux += ("        UNIFEI                       ");
    }
}

aux += ("\n");
tracos();
console.log(aux);


function espaco(){
    aux += ("\n");
    for (let i = 0; i < 39; i++) {
        if(i === 0 || i === 38){
            aux += ("|");
        }
        else{
            aux += (" ");
        }   
    }
}

function tracos(){
    for (let i = 0; i < 39; i++) {
        aux += ("-");
    }
}
